Assets/Scripts/SettingsMenu.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.Audio;
+using UnityEngine.UI;
+
+public class SettingsMenu : MonoBehaviour
+{
+
+    public AudioMixer audioMixer;
+    //public Slider volumeSlider;
```

Comment:
this is fine since you're practicing right now but in the future, when you're submitting production-level code for review, make sure any commented out code gets deleted (unless you have a specific need to leave it in, in which case you would add another comment or `// TODO` explaining why)